script/debug/loop_junit.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/bash
+
+cnt=0
+while [ true ]; do 
+    echo $cnt
+    timeout 10m python3 ../testing/junit/run_junit.py --build-type=debug --query-mode=simple | tee log_junit.log
+    test $? -ne 0 && break
+    cnt=$((cnt + 1))
+done
```

Comment:
This is your debugging scripts for the docker problem, right? If yes, then I don't think we want to include it in the repo.

Reply:
sure, i will remove the scripts in the debug folder from the git